tests/mtg.test.ts

```diff
@@ -132,8 +132,7 @@ Sideboard
 1 Tamiyo, Collector of Tales (WAR) 220
 3 Chandra, Awakened Inferno (M20) 127`;
 
-let pushTranslateData = `!push Bant V6 // standard // BO3 // Bant ! :
-Deck
+let pushTranslateData = `Deck
 3 Île (ELD) 257
```

Comment:
English

Reply:
The purpose of this test is to check if translate functions works correctly from french to english.